README.md

```diff
@@ -1,16 +1,17 @@
 # uqcsbot
 
-uqcsbot is a chat bot built on the [Hubot][hubot] framework. It was
-initially generated by [generator-hubot][generator-hubot], and configured to be
-deployed on [Heroku][heroku] to get you up and running as quick as possible.
+uqcsbot is a chat bot built on the [Hubot][hubot] framework for use on our [UQCS Slack Team](uqcs.slack.com).
 
-[heroku]: http://www.heroku.com
-[hubot]: http://hubot.github.com
-[generator-hubot]: https://github.com/github/generator-hubot
+#### Set-up in Windows (with cygwin)
+
+0. Ensure node, npm and cygwin are installed
+1. `npm install`
+2. `dos2unix bin/hubot`
+3. `bin/hubot`
 
 ### Running uqcsbot locally
 
-**If you're having trouble building/running the bot don't be afraid to ask for help in the UQCS slack**
+**If you're having trouble building/running the bot, don't be afraid to ask for help in the UQCS slack**
 
 You can test your hubot by running the following, however some plugins will not
 behave as expected unless the [environment variables](#configuration) they rely
```

Comment:
This link will be broken when the configuration section below is removed.

Reply:
good catch :^) fixed